pkg/iostreams/console_windows.go

```diff
@@ -3,14 +3,8 @@
 
 package iostreams
 
-import (
-	"golang.org/x/sys/windows"
-)
-
-func enableVirtualTerminalProcessing(fd uintptr) error {
-	stdout := windows.Handle(fd)
-
-	var originalMode uint32
-	windows.GetConsoleMode(stdout, &originalMode)
-	return windows.SetConsoleMode(stdout, originalMode|windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
+func hasAlternateScreenBuffer(hasTrueColor bool) bool {
+	// on Windows we just assume that alternate screen buffer is supported if we
+	// enabled virtual terminal processing, which in turn enables truecolor
+	return hasTrueColor
```

Comment:
Does windows not support `os.Getenv("TERM")`?

Reply:
To my knowledge, Windows typically doesn't advertise terminal features via the TERM environment variable, but it does expose them programmatically through Console APIs. Hence, if we successfully enabled virtual terminal processing via these APIs, I think we can safely assume that alternate screen buffers are enabled. This is why the check isn't symmetrical with non-Windows platforms.